src/javascript/app/Stores/Modules/Contract/contract-store.js

```diff
@@ -126,6 +129,19 @@ export default class ContractStore extends BaseStore {
         }
     }
 
+    handleSubscribeProposalOpenContract = (contract_id, cb) => {
+        const proposal_open_contract_request = [contract_id.toString(), cb, false];
```

Comment:
can you add a comment to remove this `toString()` once API starts sending contract id as a number?